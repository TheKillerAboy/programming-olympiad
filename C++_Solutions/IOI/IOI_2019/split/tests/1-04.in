7d6697fb-2a55-4d69-94ee-21b41f37da6a
4 4
2 1 1
2 1
3 2
3 0
0 1
