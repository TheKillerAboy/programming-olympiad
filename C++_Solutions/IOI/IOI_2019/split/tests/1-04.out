9e46656f-28f5-4a1b-acb6-f2309f627891
OK
1 2 3 1
